public/index.html

```diff
@@ -231,18 +231,18 @@ <h2 ng-show="right.subHeading">{{right.subHeading}}</h2>
 
             <div class="row">
                 <div class="name lanc">{{badminton.player1}}</div>
-                <div class="games">{{badminton.set1}}</div>
-                <div class="matches">{{badminton.leg1}}</div>
+                <div class="games">{{badminton.game1}}</div>
+                <div class="matches">{{badminton.match1}}</div>
             </div>
 
             <div class="row">
                 <div class="name york">{{badminton.player2}}</div>
-                <div class="games">{{badminton.set2}}</div>
-                <div class="matches">{{badminton.leg2}}</div>
+                <div class="games">{{badminton.game2}}</div>
+                <div class="matches">{{badminton.match2}}</div>
             </div>
 
             <div class="row">
-                 <div class="footer">best of 3 games wins match</div>
+                 <div class="footer">best of 3 matches wins</div>
```

Comment:
The original version of this was right - see general pull request comments where I explain why.